bayesian/api_v1.py

```diff
@@ -240,6 +240,11 @@ class ComponentAnalyses(Resource):
     def get(ecosystem, package, version):
         """Handle the GET REST API call."""
         package = urllib.parse.unquote(package)
+        if check_for_accepted_ecosystem(ecosystem) is False:
```

Comment:
`if not check_for_accepted_ecosystem(ecosystem):`

Reply:
changed